app/views/HealthIndicatorsLeaderBoard.scala.html

```diff
@@ -58,9 +71,16 @@ <h1>Health Indicator Leaderboard</h1>
                             <a href="/health-indicators/@{indicator.repoName}"><span class="repoName">@indicator.repoName</span></a>
                         </td>
                         <td id="row_@{i}_column_1">
-                            @{if(indicator.owningTeams.size == 1){indicator.owningTeams.map(_.asString)}
-                            else if(indicator.owningTeams.size <=3){indicator.owningTeams.map(_.asString).mkString(", ")}
-                            else("Multiple")}
+                            <ul class="list-group">
+                                @if(indicator.owningTeams.size > 10) {
+                                    <li title="@{indicator.owningTeams.map(_.asString).mkString(", ")}"
+                                    data-toggle="tooltip" data-placement="bottom">Multiple Teams</li>
```

Comment:
Should we use the same terminology as `DefaultBranchListPage`? "Various" or "Multiple"?

Reply:
This originally just said "Multiple" so happy to revert it back for consistency